python/in_toto_attestation/v1/statement.py

```diff
@@ -0,0 +1,36 @@
+#  Wrapper class for in-toto attestation Statement protos.
+
+import in_toto_attestation.v1.statement_pb2 as spb
+from in_toto_attestation.v1.resource_descriptor import ResourceDescriptor
+
+STATEMENT_TYPE_URI = 'https://in-toto.io/Statement/v1'
+
+class Statement:
+    def __init__(self, s=None):
```

Comment:
I'm not sure if we want to follow any particular style conventions, so I guess these comments are optional :)

It's not obvious from the method signature what `s` is.  Statements require subjects so it could conceivably be that.  Perhaps it could be given a more descriptive name (or comments)?  Alternatively, I might have expected __init__ to take the subject of the statement (since it's required).  If we want to have the ability to copy from an existing statement perhaps that could be a separate function?  Perhaps something like

```python
@staticmethod
def copy_from_pb(s):
  # ...
```